src/coreclr/vm/reflectioninvocation.cpp

```diff
@@ -279,29 +296,35 @@ class ArgIteratorForMethodInvoke : public ArgIteratorTemplate<ArgIteratorBaseFor
     }
 };
 
-FCIMPL4(Object*, RuntimeMethodHandle::InvokeMethod,
-    Object *target,
+extern "C" void QCALLTYPE RuntimeMethodHandle_InvokeMethod(
```

Comment:
FWIW there have been recent discussions about removing this and moving to emit-only plus some work for handling startup\warmup scenarios since the overhead of emit (plus JIT) is high.

Reply:
RIght. That is why I'm doing this in such a naive manner. If we see a substantial regression in the perf lab, I will claw some back for .NET 10, but I'm banking on it being removed in the short to medium term.